components/family-studies/ui/src/main/resources/PhenoTips/FamilySelector.xml

```diff
@@ -125,7 +125,7 @@
   widgets.FamilySelector = Class.create({
 
         initialize: function() {
-            console.log("init");
+            console &amp;&amp; console.log("init");
```

Comment:
Heh, I'd remove them from everywhere :) I didn't want to actually change functionality. I'll remove them from FamilySelector if you prefer. Anywhere else?